index.html

```diff
@@ -98,16 +81,17 @@
         Introduction
       </h2>
       <p>
-        To use Payment Request API, merchants (or their service providers) must
-        know how to handle payment method specific data for those methods they
-        claim to accept. This allows the API to abstract away the details of
-        specific payment methods and to add new ones over time without changing
-        the API specification.
+        To use the <a data-cite="payment-request">Payment Request API</a>,
+        merchants (or their service providers) need know how to handle payment
+        method specific data for those methods they claim to accept. This
+        allows the API to abstract away the details of specific payment methods
+        and to add new ones over time without changing the <abbr title=
+        "application programming interface">API</abbr>.
```

Comment:
Probably the first use of API in the document should get this treatment, not one middle-way through the document. Although IMO it's not necessary at all.